src/Router/AdminUrlGenerator.php

```
@@ -329,8 +331,22 @@ public function generateUrl(): string
             );
         }
 
-        $url = $this->urlGenerator->generate($routeName, $routeParameters, $urlType);
-        $url = '' === $url ? '?' : $url;
+        try {
+            $url = $this->urlGenerator->generate($routeName, $routeParameters, $urlType);
+            $url = '' === $url ? '?' : $url;
+        } catch (MissingMandatoryParametersException | RouteNotFoundException $e) {
+            // If URL generation fails because required parameters are missing or the route
+            // is not found, avoid breaking template rendering by returning a safe
+            // fallback URL. Use the dashboard route as a safe entry point. This
+            // preserves the behavior expected in templates (links are rendered)
+            // while avoiding uncaught exceptions during layout rendering.
```

Not sure to understand this. If the link is rendered with the wrong URL, what is the benefit?

---

I agree with @Seb33300. This looks like hiding the error instead of fixing it. I think it's the same issue reported in #7082. I haven't been able to reproduce it yet. I want to do that to propery fix this.